src/v/cluster/commands.h

```diff
@@ -265,21 +266,56 @@ serialize_cmd(Cmd cmd) {
             });
       });
 }
+template<typename Cmd>
+CONCEPT(requires ControllerCommand<Cmd>)
```

Comment:
I think we dropped CONCEPT in https://github.com/redpanda-data/redpanda/pull/4919